README.md

```diff
@@ -309,6 +334,39 @@ Integrate with GitHub to automate issue tracking and project management tasks. P
 4. **github_repository**: Name of the repository you want the GitHub issues to be scanned. Eg: `holmesgpt`.
 </details>
 
+<details>
+<summary>PagerDuty Integration</summary>
+
+Integrate with PagerDuty to automate issue tracking and project management tasks. Provide your PagerDuty credentials and specify the user email to update the ticket with findings.
```

Comment:
Here too, use Incident terminology and not Issue Tracking and Project Management and Ticket which is Jira terminology